### ISSUE TITLE
`@ofborg test foo` in PR description does nothing

### ISSUE THREAD

**Issue by l0b0** [NixOS/ofborg#658]
See https://github.com/NixOS/nixpkgs/pull/263295 - the "disable-installer-tools" was not scheduled.

**Comment by cole-h** [comment]
Correct. ofborg currently does not read the PR body for commands; they have to be made as a comment on the PR. I don't know how easy it would be to get ofborg to read the PR body, so for now your only option is to make a separate comment.

(Note that you can edit your comment and ofborg will pick up those edits as a new command, so if you want to re-run the test, you just need to make a no-op edit, like adding a space at the end.)

**Comment by wolfgangwalther** [comment]
Duplicate of #82.
